Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -1361,6 +1364,41 @@ def _collect(self) -> CollectionResult | None:
         ))
 
 
+class E2ETestCollector(TestCollector, ABC):
+
+    @abstractmethod
+    def get_e2e_packs(self) -> set[str]:
+        """
+        Implement this abstract method to collect relevant packs for xsoar/xsoar-saas/xsiam,
+        in the future when there will be a nightly for xsoar-saas, we will install all nightly packs including e2e tests
+        so this logic will be removed
+        """
+        pass
```

Comment:
```suggestion
        raise NotImplementedError
```
do this, that's why ruff is complaining about your code

Reply:
fixed